example/app-model/src/test/java/com/mastercard/test/flow/example/app/model/ExampleSystemTest.java

```diff
@@ -120,23 +120,23 @@ void inheritanceHealth() {
 						"│Total Debt :                    196│",
 						"├─────Actual──────┬─────Optimal─────┤",
 						"|roots         362│roots         120|",
-						"|edges         398│edges         444|",
-						"|total         760│total         564|",
-						"|        3  25.00%│        3  21.43%|",
-						"|        1   8.33%│        2  14.29%|",
-						"|        1   8.33%│        3  21.43%|",
-						"|        2  16.67%│        3  21.43%|",
-						"|        1   8.33%│        0   0.00%|",
+						"|edges         400│edges         446|",
+						"|total         762│total         566|",
+						"|        4  30.77%│        4  26.67%|",
+						"|        1   7.69%│        2  13.33%|",
+						"|        1   7.69%│        3  20.00%|",
+						"|        2  15.38%│        3  20.00%|",
+						"|        1   7.69%│        0   0.00%|",
 						"|        0   0.00%│        0   0.00%|",
-						"|        2  16.67%│        0   0.00%|",
-						"|        1   8.33%│        1   7.14%|",
+						"|        2  15.38%│        0   0.00%|",
+						"|        1   7.69%│        1   6.67%|",
 						"|        0   0.00%│        0   0.00%|",
 						"|        0   0.00%│        0   0.00%|",
-						"|        0   0.00%│        1   7.14%|",
+						"|        0   0.00%│        1   6.67%|",
 						"|        0   0.00%│        0   0.00%|",
-						"|        1   8.33%│        0   0.00%|",
+						"|        1   7.69%│        0   0.00%|",
 						"|        0   0.00%│        0   0.00%|",
-						"|        0   0.00%│        1   7.14%|",
+						"|        0   0.00%│        1   6.67%|",
 						"|        0   0.00%│        0   0.00%|",
 						"|        0   0.00%│        0   0.00%|",
 						"|        0   0.00%│        0   0.00%|",
```

Comment:
The addition of the new flow has shifted the debt histogram about a bit, but note that the Total Debt figure has not increased. This indicates that we chose the best possible derivation basis for our new flow.